fl4health/servers/nnunet_server.py

```diff
@@ -177,22 +177,25 @@ def update_before_fit(self, num_rounds: int, timeout: Optional[float]) -> None:
                 information from a client. Defaults to None, which indicates indefinite timeout.
         """
 
+        server_nnunet_plans_exist = self.fl_config.get("nnunet_plans") is not None
+
         # If the per_round_checkpointer has been specified and a state checkpoint exists, we load state
         # NOTE: Inherent assumption that if checkpoint exists for server that it also will exist for client.
         if self.per_round_checkpointer is not None and self.per_round_checkpointer.checkpoint_exists():
             self._load_server_state()
         # Otherwise, we're starting training from "scratch"
-        elif self.per_round_checkpointer is not None:
-            # If the state checkpointer is not None, then we want to do state checkpointing. So we need information
-            # from the clients in the form of get_properties.
+        elif self.per_round_checkpointer is not None or not server_nnunet_plans_exist:
```

Comment:
if per round checkpointer is None, does the server model need to be initialized? It will be if server_nnunet_plans_exist is True

Reply:
Yes, that's a good catch. Apologies for missing that.